main.cpp

```diff
@@ -52,106 +52,114 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
+//  action 2: the speaker light change color
+speaker.changeLightColor();
+//  action 3: the speaker shut down
+speaker.shutDown();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: alarmClock
+//  action 1: the clock is ringing
+alarmClock.ring();
+//  action 2: the clock is repeat every 5 min
+alarmClock.repeat();
+//  action 3: the clock is stop ringing
+alarmClock.stop();
+
+//  5) 
+//  Noun: fan
+//  action 1: the fan is spinning 
+fan.spin();
+//  action 2: the fan blows cold air
+fan.blowColdAir();
+//  action 3: the fan blows hot air
+fan.blowHotAir();
+  
+//  6) 
+//  Noun: computer
+//  action 1: computer get into sleep mode
+computer.sleepMode();
+//  action 2: computer backup your data
+computer.backupData();
+//  action 3: computer fans starts working
+computer.fansWorking();
 
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: cleaningRobot
+//  action 1: start to clean the house
+cleaningRobot.startCleanHouse();
+//  action 2: return to the charging station
+cleaningRobot.returnChargeStation();
+//  action 3: beep when the tank is full
+cleaningRobot.tankFullBeep();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: smokeDetector
+//  action 1: beep when sense smoke
+smokeDetector.senseSmokeBeep();
+//  action 2: call the firemans
+smokeDetector.callFiremans();
+//  action 3: drops water to put of the fire
+smokeDetector.dropWater();
+
+//  9) 
+//  Noun: traficLight
+//  action 1: the Light is red
+traficLight.Red();
+//  action 2: the Light is yellow
+traficLight.Yellow();
+//  action 3: the Light is green
+traficLight.Green();
+
 //  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-
-
+//  Noun: dog
+//  action 1: dog is barking
+dog.bark();
+//  action 2: dog is sleeping
+dog.sleep();
+//  action 3: dog is running
+dog.run();
 #endif
 
 /*
  MAKE SURE YOU ARE NOT ON THE MASTER BRANCH
 
- Commit your changes by clicking on the Source Control panel on the left, entering a message, and click [Commit and push].
- 
- If you didn't already: 
+ Commit your changes by clicking on the Source Control panel on the left,
+ entering a message, and click [Commit and push].
+
+ If you didn't already:
     Make a pull request after you make your first commit
-    pin the pull request link and this repl.it link to our DM thread in a single message.
+    pin the pull request link and this repl.it link to our DM thread in a single
+ message.
 
  send me a DM to review your pull request when the project is ready for review.
 
  Wait for my code review.
  */
 
 #include <iostream>
-int main()
-{
-    std::cout << "good to go" << std::endl;
-    return 0;
+int main() {
+  std::cout << "good to go" << std::endl;
+  return 0;

```

Comment:
Undo this change to the formatting that you made
Do not change code that you didn't write unless you are told to change it.
- [ ] revise